app/views/home_student/show.html.erb

```diff
@@ -5,25 +5,36 @@
  %>
 
  <% if current_user %>
-  <div id='student-status-main' class='main student-status'>
-    <h1><%= @user.name %></h1>
-    <h2>Posts</h2>
-      <%= render 'post_nav_links' %>
-      <br><br>
-      <%= render 'posts/post_index' %>
-      <%= render 'post_nav_links' %>
-  </div>
-  <div id='student-status' class='sidebar'>
-    <h2><%= Course.find(@user.sections[0].course_id).title %></h2>
-    <ul>
-      <% @assessments.order(due_date: :desc).each do |a| %>
-        <li class='student-status <%= annunciator(a) %>'><span class='student-status'>
-          <%= link_to a.title,
-            new_or_edit(current_user, a)
-          %><br>
-          Due: <%= a.due_date.to_date.to_s(:long) %>
-        <span></li>
-      <% end %>
-    </ul>
-  </div>
+ <div class='row'>
+   <div class='col-md-8'>
+     <div id='student-status-main' class='main student-status'>
```

Comment:
Do you mean eliminating this line " <div id='student-status-main' class='main student-status'>" and the similar line in the sidebar? Those are there to provide styling hooks for the two areas. Any reason to remove them – other than just reducing clutter?